raaghu-components/src/rds-comp-side-navigation/rds-comp-side-navigation.tsx

```
@@ -14,10 +14,11 @@ export interface RdsCompSideNavigationProps {
   toggleClass?: any;
   collapse?: any;
   logo?: string;
+  lockIconVisible?: boolean;
 }
 
 const RdsCompSideNavigation = (props: RdsCompSideNavigationProps) => {
-  return <RdsSideNav logo={props.logo} sideNavItems={props.sideNavItems} toggleTheme={props.toggleTheme} collapse={props.collapse} toggleClass={props.toggleClass}></RdsSideNav>;
+  return <RdsSideNav logo={props.logo} sideNavItems={props.sideNavItems} toggleTheme={props.toggleTheme} collapse={props.collapse} toggleClass={props.toggleClass} lockIconVisible></RdsSideNav>;
```

ESLint

[error] Expected indentation of 4 spaces but found 2.

(indent)